src/classifier/Features.h

```diff
@@ -50,4 +55,76 @@ class Features {
   Pipe *pipe_; // The pipe that owns this feature handler.
 };
 
+
+//struct  FeatureLabelPair : Structure to define a a feature-label pair
+struct FeatureLabelPair {
+	uint64_t feature;
+	int label;
+};
+
+//struct  FeatureLabelPairMapper: Structure to define a hash function and a comparison function for FeatureLabelPair
```

Comment:
Don't write lines over 80 chars. You can configure your editor to display a right margin at line 80.